eZ/Bundle/EzPublishRestBundle/Resources/config/input_parsers.yml

```diff
@@ -317,7 +317,7 @@ services:
 
     ezpublish_rest.input.parser.URLWildcardCreate:
         parent: ezpublish_rest.input.parser
-        class: "%ezpublish_rest.input.parser.URLWildcardCreate.class%"
+        class: "%ezpublish_rest.input.parser.UrlWildcardCreate.class%"
```

Comment:
based on service name it seems rather param should be renamed, or?

Reply:
Hah, yes, but then it could be considered a BC break? I'll let you decide :)

Reply:
But considering that using `*.class` parameters goes against best practices and should just be removed, I would be fine with keeping it this way too.

Reply:
@alongosz any pov?

Reply:
For `6.7` it was a bug as actual parameter name is `ezpublish_rest.input.parser.UrlWildcardCreate.class`, so I'm fine with this change.

In the future we probably should drop `.class` parameters.